pages/docs/reference/js-ir-compiler.md

```diff
@@ -0,0 +1,65 @@
+---
+type: doc
+layout: reference
+category: "JavaScript"
+title: "Using the experimental IR compiler"
+---
+# Using the experimental Kotlin/JS IR compiler
+
```

Comment:
Add a note about the experimental status like the one on [this page](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html)